src/main/webapp/app/clinical/clinical.component.ts

```diff
@@ -48,13 +48,12 @@ export class ClinicalComponent implements OnInit {
             const ageGroups = this.clinicalInput.age_numerical.split(',');
             // Age input cannot only accepts age range groups greater than 2.
             if (ageGroups.length === 2) {
-                const ageNumber0 = Number(ageGroups[0].match(/\d\d?$/));
-                const ageNumber1 = Number(ageGroups[1].match(/\d\d?$/));
+                const ageNumber = this.clinicalInput.age_numerical.match(/\d+(\.?\d+)?/g).map(function(v) { return parseFloat(v); });
```

Comment:
Why don't we just use lodash _.isNumber?

Reply:
_.isNumber() returns boolean value. I need to convert string to float number.

Reply:
You just need to know whether `clinicalInput.age_numerical` is number then you can just converted a string a number by doing Number(string)